pyfr/plugins/base.py

```diff
@@ -155,3 +155,16 @@ def _prepare_region_data_eset(self, intg, eset):
 
             self._ele_regions.append((doff, etype, darr))
             self._ele_region_data[f'{etype}_idxs'] = darr
+
+    def prepare_surface_data(self, intg):
```

Comment:
Another possibility is that we instead save the region instance, or if the region is '*' don't set it.  This will make it easy to query faces later on should someone need them.  Plus callers can then use isinstance to see if the region is geometric or not.